fr/migrations.rst

```diff
@@ -305,212 +399,124 @@ crée le fichier::
         }
     }
 
-Les noms des migrations peuvent suivre l'un des motifs suivants:
-
-* Créer une table: (``/^(Create)(.*)/``) Crée la table spécifiée.
-* Supprimer une table: (``/^(Drop)(.*)/``) Supprime la table spécifiée. Ignore les arguments de champ spécifié.
-* Ajouter un champ: (``/^(Add).*(?:To)(.*)/``) Ajoute les champs à la table spécifiée.
-* Supprimer un champ: (``/^(Remove).*(?:From)(.*)/``) Supprime les champs de la table spécifiée.
-* Modifier une table:  (``/^(Alter)(.*)/``) Modifie la table spécifiée. Un alias pour CreateTable et AddField.
-
-Les types de champs sont ceux mis à disposition par la bibliothèque `` Phinx``.
-Cela peut être:
-
-* string
-* text
-* integer
-* biginteger
-* float
-* decimal
-* datetime
-* timestamp
-* time
-* date
-* binary
-* boolean
-* uuid
-
-De plus, vous pouvez créer un fichier migrations vide si vous voulez un contrôle
-total sur ce qui doit être exécuté::
-
-    bin/cake migrations create MyCustomMigration
-
-Prenez soin de lire la documentation officielle Phinx
-`<http://docs.phinx.org/en/latest/migrations.html>` _ afin de connaître la liste
-complète des méthodes que vous pouvez utiliser pour écrire des fichiers de
-migration.
-
 Générer une Migration à partir d'une Base de Données Existante
---------------------------------------------------------------
+==============================================================
 
 Si vous avez affaire à une base de données pré-existante et que vous voulez
 commencer à utiliser migrations, ou que vous souhaitez versionner le schéma
 initial de votre base de données, vous pouvez exécuter la commande
 ``migration_snapshot``::
 
-    bin/cake bake migration_snapshot Initial
+    $ bin/cake bake migration_snapshot Initial
 
 Elle va générer un fichier de migration appelé **Initial** contenant toutes les
 déclarations pour toutes les tables de votre base de données.
 
-Créer des Clés Primaires Personnalisées
----------------------------------------
-
-Pour personnaliser la création automatique de la clé primaire ``id`` lors
-de l'ajout de nouvelles tables, vous pouvez utiliser le deuxième argument de la
-méthode ``table()``::
-
-    <?php
-
-    use Migrations\AbstractMigration;
-
-    class CreateProductsTable extends AbstractMigration
-    {
-        public function change()
-        {
-            $table = $this->table('products', ['id' => false, 'primary_key' => ['id']]);
-            $table
-                  ->addColumn('id', 'uuid')
-                  ->addColumn('name', 'string')
-                  ->addColumn('description', 'text')
-                  ->create();
-        }
-    }
-
-Le code ci-dessus va créer une colonne ``CHAR(36)`` ``id`` également utilisée
-comme clé primaire.
-
-.. note::
+Par défaut, le snapshot va être créé en se connectant à la base de données
+définie dans la configuration de la connection ``default``.
```

Comment:
"connexion".
But since we are talking about a config of the Connection object, we might need to find a better way of talking about this.